util/CMakeLists.txt

```diff
@@ -397,6 +397,19 @@ endif()
 # Copied from: https://github.com/qedsoftware/crashpad/blob/3583c50a6575857abcf140f6ea3b8d11390205b3/util/CMakeLists.txt#L196-L233
 if(APPLE)
     if(NOT IOS)
+        if(NOT CMAKE_OSX_SYSROOT)
+            # In CMake 4.0, CMAKE_OSX_SYSROOT defaults to empty unless SDKROOT is set:
+            # https://cmake.org/cmake/help/latest/variable/CMAKE_OSX_SYSROOT.html
+            execute_process(
+                COMMAND xcrun --sdk macosx --show-sdk-path
+                RESULT_VARIABLE CMAKE_OSX_SYSROOT_RESULT
+                OUTPUT_VARIABLE CMAKE_OSX_SYSROOT
+                OUTPUT_STRIP_TRAILING_WHITESPACE
+            )
+            if(NOT CMAKE_OSX_SYSROOT_RESULT EQUAL 0)
+                message(FATAL_ERROR "Failed to resolve CMAKE_OSX_SYSROOT. Please specify SDKROOT in the environment.")
```

Comment:
The `FATAL_ERROR` here should indicate that we couldn't find an SDK for `macosx` (and display the result value of `xcrun`).